### PR TITLE
Add: Login, Registration and Forgot Password Screen Designs

### PR DESCRIPTION
**Pull request by parthsolankisimform**
Added Modules for the Following Screens: 
- Login Module
- Registration Module
- Forgot Password Module

### CODE REVIEW

File: app/src/main/java/simform/gitexcercise/android/module/authentication/forgotPassord/ForgotPasswordActivity.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            doValidatePassword()
+        }
+    }
+
+    private fun doValidatePassword() {
+        if (binding.textForgotPasswordTextEditText.text.isNullOrBlank()) {
+            binding.textForgotPasswordTextInput.apply {
+                isErrorEnabled = true
+                error = getString(R.string.text_error_email)
+                Handler(Looper.getMainLooper()).postDelayed(5000) {
```
Comment (yashwantgowla-simform) [inline app/src/main/java/simform/gitexcercise/android/module/authentication/forgotPassord/ForgotPasswordActivity.kt]: don't use hardcoded values

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        app:placeholderText="@string/msg_enter_email_address">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/textForgotPasswordTextEditText"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:hint="@string/hint_enter_email_address"
+            android:inputType="textEmailAddress"
+            android:maxLines="@string/_1"
+         />
```
Comment (yashwantgowla-simform) [inline app/src/main/res/layout/activity_forgot_password.xml]: code should be formatted.

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        android:layout_width="@dimen/_0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/msg_forgot_password"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/textForgotPasswordTextInput"
+        app:layout_constraintStart_toStartOf="@id/textForgotPasswordTextInput"
+        app:layout_constraintTop_toBottomOf="@id/imageCompanyLogo"
+        android:textAlignment="center" />
+
+
```
Comment (yashwantgowla-simform) [inline app/src/main/res/layout/activity_forgot_password.xml:30]: remove unnecessary lines

File: app/src/main/java/simform/gitexcercise/android/module/authentication/forgotPassord/ForgotPasswordActivity.kt (hunk 1 above)
Comment (parthsolankisimform) [reply app/src/main/java/simform/gitexcercise/android/module/authentication/forgotPassord/ForgotPasswordActivity.kt]: Resolved

File: app/src/main/res/layout/activity_forgot_password.xml (hunk 2 above)
Comment (parthsolankisimform) [reply app/src/main/res/layout/activity_forgot_password.xml]: Resolved

File: app/src/main/res/layout/activity_forgot_password.xml (hunk 3 above)
Comment (parthsolankisimform) [reply app/src/main/res/layout/activity_forgot_password.xml:30]: Resolved

**Review by yashwantgowla-simform — APPROVED** [review]

**Pull request merged**